src/components/Calendar/EventModal.tsx

```diff
@@ -18,6 +19,10 @@ const dateFormatOptions: Intl.DateTimeFormatOptions = {
     day: "numeric",
 };
 
+const StyledParagraph = styled.p`
```

Comment:
Would it make sense to have `StyledParagraph` be specified in `themes` or `globalStyles` instead of just in `EventModal` to standardize the style of "normal text" for each page? (Same for h1, h2, etc). Though that might be for another ticket.

Reply:
I added this because I would like the padding between different lines in the event modal to be larger, and the padding is adjusted specifically based on the styling of the modal. It is probably not that reusable in other parts mmm

Resonating to your idea though, there are some styling standardized for h1 h2 and p though (especially the font), and there is also a reusable Title component made for different pages if it is useful!